README.md

```diff
@@ -58,13 +63,15 @@ From here we will build the image for the example resource driver:
 This also makes the built images available to the `kind` cluster.
 
 We now install the NVIDIA GPU DRA driver:
-```
+```bash
 ./demo/clusters/kind/install-dra-driver.sh
 ```
 
 This should show two pods running in the `nvidia-dra-driver` namespace:
-```
+```console
 $ kubectl get pods -n nvidia-dra-driver
+```
+```
```

Comment:
What is the purpose of separating this if you keep the `$`?

Reply:
A user can copy/paste and run the command directly.

Reply:
A user can copy/paste and run an example command directly.

Reply:
I'm trying to say that you didn't remove the `$` in your diff.

Reply:
Removed it.